From 32affe610606b3a550c2953993a72063eb2b7381 Mon Sep 17 00:00:00 2001
From: Fabian Bornschein <fabiscafe@mailbox.org>
Date: Mon, 18 Sep 2023 23:31:59 +0200
Subject: [PATCH] rm unused and problematic .icon-theme.cache

This file was probably inserted by mistake. It serves no purpose.
---
 Adwaita/.icon-theme.cache | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 delete mode 100644 Adwaita/.icon-theme.cache

diff --git a/Adwaita/.icon-theme.cache b/Adwaita/.icon-theme.cache
deleted file mode 100644
index e69de29bb..000000000
-- 
GitLab

